RELEASE_NOTES.rst

```diff
@@ -9,6 +9,17 @@ All changes
 - Adjust the Westeros reporting tutorial to pyam 1.0 deprecations (:pull:`492`).
 
 
+.. _v3.4.0:
+
+v3.4.0
+======
+
+All changes
+-----------
+
+- Add Westeros :doc:`tutorial <tutorials>` on historical parameters (:pull:`478`).
```

Comment:
This single added line should appear next to the 3 existing lines under "Next release / all changes"; there should be no other additions to this file. Specific version numbers are **only** added to this file when a release is made.

Reply:
Done